mattermod.go

```diff
@@ -23,21 +25,16 @@ func main() {
 
 	config, err := server.GetConfig(configFile)
 	if err != nil {
-		err = errors.Wrap(err, "unable to load server config")
-		mlog.Error("unable to load server config", mlog.Err(err))
-		panic(err)
+		mlog.Error("unable to load server config", mlog.Err(errors.Wrap(err, "unable to load server config")))
```

Comment:
Wondering if it makes sense to just do: `mlog.Error("unable to load server config", mlog.Err(err))`? The string is repeated, and we are not really propagating the error upwards.